torch/quantization/observer.py

```diff
@@ -298,7 +299,10 @@ def __init__(self, dtype=torch.quint8, qscheme=torch.per_tensor_affine,
 
     def forward(self, x_orig):
         r"""Records the running minimum and maximum of ``x``."""
-        x = x_orig.detach()
+        return self._forward(x_orig)
```

Comment:
Why do we need this?

Reply:
oh, we probably don't need this, I was trying to ignore _forward previously, but it turns out that it doesn't work.